### ISSUE TITLE
Need help using PoseGroup with animated Items

### ISSUE THREAD

**Issue by JoaoCnh** [Popmotion/popmotion#283]
I'm trying to use PoseGroup to animate the opacity of the items just like the examples.
But my items already have poses that are triggered on hover.

Basically

```
<PoseGroup animateOnMount={true}>
  {items.map(item => <Item key={item.id} />
</PoseGroup>
```

In the Item I have:

```
const Item = posed.div({
  enter: { opacity: 1 },
  exit: { opacity: 0 },
  idle: { scale: 1 },
  hovered: { scale: 1.5 },
});
```

Well to handle the hover state properly I must set a pose right?

`<Item pose={isHovered ? "hovered" : "idle"} />`

The enter and exit animations are not played on mount. When the list first appears.

How can I achieve this?

**Comment by InventingWithMonster** [comment]
Looks like a bug to me. Let me take a look

**Comment by InventingWithMonster** [comment]
Can you try this now 1.1.3 has been released and also point me to a codepen or full example? 

Thanks

**Comment by somebody32** [comment]
@InventingWithMonster I have the same problem. Here you have 2 sandboxes to see it in action:

[Without PoseGroup](https://codesandbox.io/s/64mqy6jm4k): try to add items, try to flip them, it works.
[Same but with PoseGroup](https://codesandbox.io/s/mjll62y469): now adding/removing children is animated but there is no way to pass `flipped` pose.

Or am I doing something wrong?

**Comment by InventingWithMonster** [comment]
@somebody32 Fixed in `1.6.1`
